.github/workflows/greetings.yml

```diff
@@ -2,6 +2,8 @@ name: Greetings
 
 on: [pull_request, issues]
 
+permissions: write-all
```

Comment:
Is this the lowest-scope permission that will solve this problem? I'd like to limit write access as much as humanly possible

Reply:
I'm referring to this [block](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defining-access-for-the-github_token-scopes) of the documentation. Please only give write access to the specific attribute

Reply:
resolved with permission changes